lib/protocol/src/openctx-protocol.schema.json

```diff
@@ -140,8 +176,49 @@
           "additionalProperties": false,
           "properties": {
             "mentions": {
-              "description": "Whether the provider support mentions.",
-              "type": "boolean"
+              "description": "Configuration for the mentions feature.",
+              "type": "object",
+              "additionalProperties": false,
+              "properties": {
+                "implements": {
```

Comment:
Can we remove this? If there is an object here, it is implemented.

```
const implementsMentions = !!meta.features.mentions
```

Ie when would it ever be false?

Reply:
I thought `!!{} = false`, but it isn't. I'll remove this.

Reply:
done